Makefile

```diff
@@ -45,7 +46,7 @@ all:  ## Build binaries for all available architectures
 	make crossbuild GOOS=windows GOARCH=386
 
 .PHONY: build_local
-build_local: ## Build local binaries without providing specific GOOS/GOARCH
+build_local: web ## Build local binaries without providing specific GOOS/GOARCH
```

Comment:
I am afraid that it will break auto-upgrade on so many existed one-click cloud image and auto-script from community.

How about let the `make web` as an independent make target ?
Mostly node's owner would not expected exposed their webUI on public internet. Those folks who wants to use webUI in sub-net should indicated `make web` explicitly.

webUI should disable by default not only launch cmd but also build process.

Reply:
I think it's better to build webui when using `make` but just not enable it by default. This is the most convenient way. As for the compatibility, as long as `make` go through it should be compatible with existing nodes (including one-click), and we can test that in advance to make sure it works.